README.md

```diff
@@ -485,6 +485,21 @@ class TritonPythonModel:
         return responses
 ```
 
+Starting from 23.09, `pb_utils.TritonError` may be constructed with an optional
+Triton error code on the second parameter. For example:
+
+```
+pb_utils.TritonError("The file is not found", pb_utils.TritonError.NOT_FOUND)
+```
+
+Supported error codes:
```

Comment:
Does there need to be more description with these error codes or is it just a 1-to-1 mapping from the triton internals?